content/blog/blog-post-05.md

```diff
@@ -0,0 +1,100 @@
+---
+authors: ["kokkos-team"]
+title: Kokkos User Group Meeting 2023
+date: "2024-01-17"
+tags: ["blog", "KUG"]
+thumbnail: img/kokkos-blog-post-5_a.jpg
+---
+
+## In a nutshell
+
+We held the second Kokkos User Group meeting in Albuquerque from December 12th-15th, 2023.
+Because of the pandemic, this was almost five years since we had the first one.
+
+### Day 0
+
+The Kokkos team got together, some of us meeting in person for the first time!
+It was great to finally put names to faces.
+
+{{< image src="img/kokkos-blog-post5_b.jpg" style="float: center; height=10">}}
+
+### Day 1
+
+This was our time to meet the community and for the community to meet us.
+
+The morning started out with updates on the state Kokkos Core and Kokkos Kernels on various exascale architectures,
+followed by build system updates to Kokkos Core.
+
+One big topic is the future of Kokkos, now that ECP is over.
+As announced at [SC23](https://sc23.supercomputing.org),
+the Linux Foundation is forming the
+[High Performance Software Foundation](https://www.linuxfoundation.org/press/linux-foundation-announces-intent-to-form-high-performance-software-foundation-hpsf)
+with Kokkos being one of the initial projects being transferred over to it.
+We are still figuring out how the project is going to be governed.
+
+After a break we went over Kokkos Kernels updates, covering batched algorithms, ML/IR, mixed precision and GEMMA.
+
+During the afternoon the first set of community talks were presented.  These included:
+* CExA (GPU computing at CEA)
```

Comment:
Do we link to the corresponding slides for each talk?

Reply:
@cedricchevalier19 I started doing that, but it was forcing me to download the PDFs instead of displaying them inside the browser.  If someone knows how to change that behavior (or folks think that behavior is fine), I can put them in.

Reply:
As discussed, we can do this at a later time.